src/components/app/grepSSEworker.ts

```diff
@@ -32,7 +35,8 @@ export default () => {
                     bufferedLogs = [log]
                 } else {
                     const { _args, a = 0, b = 0 } = this.grepTokens[0]
-                    if (new RegExp(_args, 'gi').test(log)) {
+
+                    if (new RegExp(_args, 'gi').test(log.replace(ANSI_COLOR_ESCAPE_SEQUENCE_REGEX, ''))) {
```

Comment:
i see us doing it at 3 different places.. should we do it once right after we get the message in case grep command is provided?

Reply:
As discussed, this change may increase the dev/testing scope as trailing lines also come into the picture. We'll update it as required along with escape characters change.